backend/app/processing/db_detector.py

```
@@ -0,0 +1,52 @@
+                # MySQL binlog
+                if len(header) >= 4 and header[:4] == b"\xfe\x62\x69\x6e":
+                    result["db_type"] = "mysql_binlog"
+                    return result
+                # SQL dump (MySQL or Postgres)
+                if b"CREATE TABLE" in header or b"INSERT INTO" in header or b"COPY " in header:
+                    if b"PostgreSQL database dump" in header:
+                        result["db_type"] = "postgres_dump"
+                    else:
+                        result["db_type"] = "sql_dump"
+                    return result
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)